releases.rst

```diff
@@ -0,0 +1,73 @@
+Releases & version support
+==========================
+
+Behat follows `Semantic Versioning`_ - breaking changes will only be made in a major release.
+
+======= ========== ============ ============ =====================================================================
+Major   Released   Bugfix EOL   Security EOL
+======= ========== ============ ============ =====================================================================
+`v2.x`_ July 2011  June 2015    June 2015    `Changelog <https://github.com/Behat/Behat/blob/2.5/CHANGES.md>`__
+`v3.x`_ April 2014 *>12 months* *>24 months* `Changelog <https://github.com/Behat/Behat/blob/master/CHANGES.md>`__
```

Comment:
This may be a bit confusing, some users may read this as "bugfixes ended 12 months after April 2014 and security fixes ended 24 months after April 2014"? Maybe adopt what other tools do which to show which versions are maintained, which are security fixes only, which are end of life and even show future unreleased versions. So we could list version 2.x as end of life for bugs and security, 3.x as maintained and 4.x as unreleased. What do you think?

Reply:
@carlos-granados thanks for the review, and yes, I wondered if that would be clear enough to a new reader.

I've reworked the structure of the page to separate out the current from unmaintained versions, I think it's easier to follow now, let me know what you think.

I haven't listed 4.x yet as I don't think there's actually anything really on it yet, and it is out of date with 3.x. We can add it when we start working on it properly.